action_msgs/msg/GoalInfo.msg

```diff
@@ -0,0 +1,6 @@
+# Goal ID
+# TODO(jacobperron): Use a ported version of https://github.com/ros-geographic-info/unique_identifier/blob/master/uuid_msgs
```

Comment:
Given that `uuid_msgs` has just a single message, I think the UUID.msg could live in `builtin_interfaces` rather than porting a package named `uuid_msgs`. Porting the tools in `unique_id` wouldn't need to block that.